lib/protobuf.ex

```diff
@@ -282,7 +282,25 @@ defmodule Protobuf do
   """
   @spec load_extensions() :: :ok
   def load_extensions() do
-    Protobuf.Extension.__cal_extensions__()
+    for mod <- get_all_modules(),
```

Comment:
Refactor: I just moved the `__cal_extensions__/0` function inline here, not sure why we kept it separate in another module.